### DISCUSSION TITLE
'BaseCase' object has no attribute 'uc_open_with_reconnect'

### DISCUSSION THREAD

**Discussion by joako233** [seleniumbase/SeleniumBase#3090] [General]
When I run my script this message appears: 'BaseCase' object has no attribute 'uc_open_with_reconnect'

```
from seleniumbase import SB

with SB(uc=True,pls='none',test=True) as sb:
    sb.maximize_window()
        
    sb.driver.default_get("https://www.reuters.com/account/sign-in/?redirect=https%3A%2F%2Fwww.reuters.com%2Fbusiness%2F")
    try:
        sb.uc_open_with_reconnect('https://www.reuters.com/',3)
    except Exception as e:
        print(e)
    input()
        

```

**Comment by joako233** [comment]
> When I run my script this message appears: 'BaseCase' object has no attribute 'uc_open_with_reconnect'
> 
> ```
> from seleniumbase import SB
> 
> with SB(uc=True,pls='none',test=True) as sb:
>     sb.maximize_window()
>         
>     sb.driver.default_get("https://www.reuters.com/account/sign-in/?redirect=https%3A%2F%2Fwww.reuters.com%2Fbusiness%2F")
>     try:
>         sb.uc_open_with_reconnect('https://www.reuters.com/',3)
>     except Exception as e:
>         print(e)
>     input()
>         
> ```
I RESOLVED

**Comment by mdmintz** [comment]
That would happen if you're not using `uc=True`. Need `uc=True` to get the UC Mode methods.

**Reply by joako233** [reply]
Yes I solved, thanks Michael
